rqrcode.gemspec

```
@@ -17,6 +17,10 @@ Gem::Specification.new do |spec|
   EOF
   spec.homepage = "https://github.com/whomwah/rqrcode"
   spec.license = "MIT"
+  spec.metadata = {
+    "bug_tracker_uri" => "https://github.com/whomwah/rqrcode/issues",
+    "changelog_uri"   => "https://github.com/whomwah/rqrcode/blob/master/CHANGELOG.md"
+  }
 
```

Caught by the linter:

```suggestion
spec.metadata = {
  "bug_tracker_uri": "https://github.com/whomwah/rqrcode/issues",
  "changelog_uri": "https://github.com/whomwah/rqrcode/blob/master/CHANGELOG.md"
}
```